k8s/deployments/mariadb.yaml

```diff
@@ -33,26 +21,14 @@ spec:
                   name: mpm-secrets
                   key: DB_PASSWORD
             - name: MYSQL_DATABASE
-              value: micro_power_manager
+              value: database-1
+            - name: MYSQL_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: mpm-secrets
```

Comment:
Could you please also update the secret's sample file? https://github.com/EnAccess/micropowermanager/blob/main/k8s/secrets/mpm-secrets.yaml.example